### DISCUSSION TITLE
Welcome to Steam-Library-Manager Discussions!

### DISCUSSION THREAD

**Discussion by mertcreates** [RevoLand/Steam-Library-Manager#90] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Watcher-Guru3D** [comment]
Hello RevoLand


Question: Where is the configuration file / registry settings located?
Problem:  I can delete the parent folder on my hard drive the holds the Steam Library Manager.exe file, SLM.settings file and the .slmcache folder.

When I run the Steam Library Manager.exe from a different / new location, it picks up my previous selections.
I would like to start the program like it has never been run on my computer before.

I am unable to locate the location of the Registry entry or is it a settings file?

Regards

**Comment by Watcher-Guru3D** [comment]
Bump

**Comment by mertcreates** [comment]
Hello @Watcher-Guru3D ,

SLM only uses the SLM.settings file for the user configurations, which selection is still remains after you remove the file or open the binary from a different folder?

**Reply by Watcher-Guru3D** [reply]
Hello RevoLand

The setting that remains are the location of the Steam library and if you would like to start the Steam game using SLM or not.

Regards

**Reply by mertcreates** [reply]
Hey,

The locations of the Steam libraries are populated directly from the Steam itself.
- Getting the Steam installation directory from registry ( https://github.com/RevoLand/Steam-Library-Manager/blob/master/Source/Steam%20Library%20Manager/Functions/Steam.cs#L26 )
- If the steam installation directory and the config.vdf file exists, read the library list directly from steam configuration itself ( https://github.com/RevoLand/Steam-Library-Manager/blob/master/Source/Steam%20Library%20Manager/Functions/Steam.cs#L674 )

Please let me know if you still have questions.
Best Regards

**Reply by Watcher-Guru3D** [reply]
Hello RevoLand

Lets try this again.  FYI:  I have my Steam Library on a separate hard drive, not on the OS drive and not in Program Files Directory.

The first time I launched SLM, I had to point SLM to where the Steam Library files were located. The first time I started SLM, I selected not to launch Steam games.

Now I can delete the directory with everything that SLM creates.  I can launch SLM in a new folder in a different location and a different folder name and SLM still knows where my Steam library is and does not present the option to have the SLM launch the Steam games. 

How do I reset SLM so it present me with all the options. I want to be able to pick the option as if I was launching SLM for the first time
I should need to select the Steam Library location and the option to launch Steam games or not? 

Regards
